root/usr/share/fdi/facts/openlldp.rb

```diff
@@ -82,7 +79,7 @@ def grep_v(cond)
             when 'chassisID'
               output.split("\n").each do |line|
                 ether = $1 if line.match(/MAC:\s+(.*)/)
-                result = Facter::Util::Macaddress.standardize(ether)
+                result = ether&.tr('-', ':')&.downcase
               end
```

Comment:
Facter 4 no longer provides the Util class, so let's find list of devices a different way.